packages/react/src/components/ToggleGroup/ToggleGroupItem/ToggleGroupItem.tsx

```diff
@@ -12,10 +13,8 @@ export type ToggleGroupItemProps = {
   /** The value of the ToggleGroupItem. If not set, the string value of the items children will be used. */
   value?: string;
   /** Icon to be displayed on the ToggleGroupItem */
-  icon?: React.ReactNode;
-  /** The text to be displayed on the ToggleGroupItem */
-  children?: string;
-} & Omit<ButtonHTMLAttributes<HTMLButtonElement>, 'value' | 'children'>;
+  icon?: ButtonProps['icon'];
+} & Omit<ButtonHTMLAttributes<HTMLButtonElement>, 'value'>;
```

Comment:
Looking at this, the props should just have been `type ToggleGroupItemProps = ButtonProps`, since its casted to be `Button`, but we'll fix that after `asChild` is introduced. Lets not forget!

Reply:
#1438